Python/Product/Debugger/Debugger/LegacyDebuggerProtocol.cs

```diff
@@ -405,6 +406,10 @@ public sealed class ModuleLoadEvent : Event {
 
             public int moduleId;
             public string moduleFileName;
+            [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+            public string moduleName;
+            [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+            public int isStdLib;
```

Comment:
bool?

Reply:
Kept blowing up in deserialization for some reason I couldn't figure out. I think the JSON serialization is more forgiving for missing/null int than bool.

Reply:
Tried changing it back to `bool` and it worked, so I guess I messed something up last time.